### DISCUSSION TITLE
What's the cost-formation model of using RGB?

### DISCUSSION THREAD

**Discussion by UkolovaOlga** [LNP-BP/presentations#35] [Q&A]
### TL;DR
RGB protocol leverages the same fees that are present in Lightning Network, not adding any additional fee to them. If you use RGB, you'll need to pay same fees as you normally do while transaction in Bitcoin.

### TS;WM

**Today there are 2 kinds of fees in LN:**
1. The first is basically the _fee of the transaction_ itself, for it being published. This fee is attached to every transaction and this is a part of the channel that you open. It can be paid by you, by the other party, or being shared between you both. 

This principle and this fee stays in place for RGB transactions.

2. The second is the _routing fee_ and it is paid when you do assets (satoshis in this case) routing. 

This fee also remains the same in RGB, but with an additional option of paying it not only in bitcoin/satoshis, but also in the RGB asset that you are doing this routing of.

**RGB DEX routing fee**

Important to notice that _on top_ of the RGB protocol there will be DEX functionality added, that would allow you to exchange one RGB asset for another. Thus, each node that is willing to do the exchange will be able to set up a fee for its exchange services and you as a user that need to will need to either accept and pay it or find a different node with a different fee. Consequently, if you are a node operator, this gives you an opportunity to monetise your node.
